chia/rpc/full_node_rpc_client.py

```diff
@@ -184,6 +184,13 @@ async def get_block_records(self, start: int, end: int) -> List:
         # TODO: return block records
         return response["block_records"]
 
+    async def get_block_spends(self, header_hash: bytes32) -> List:
+        try:
+            response = await self.fetch("get_block_spends", {"header_hash": header_hash.hex()})
+        except Exception:
```

Comment:
This should not catch, if it fails, we should return None or allow the caller to handle the exception

Reply:
Opted to return null as some other RPC calls are doing that.